src/presentation/http/router/note.test.ts

```diff
@@ -518,6 +518,299 @@ describe('Note API', () => {
 
       expect(response?.json().message).toStrictEqual(expectedMessage);
     });
+
+    test('Returns two parents in case of relation between child and parent notes with 200 status', async () => {
+      /** Create test user */
+      const user = await global.db.insertUser();
+
```

Comment:
since only assertions and expected parent structure changes in this block of tests, we should use test.each pattern (we already have it in `noteList.test.ts`, you can check)